themes/book/app/components/SidebarFooter.tsx

```diff
@@ -0,0 +1,15 @@
+import type { GenericParent } from 'myst-common';
+import { MyST } from 'myst-to-react';
+import { MadeWithMyst } from '@myst-theme/icons';
+
+export function SidebarFooter({ content }: { content?: GenericParent }) {
+  if (!content) {
+    return <MadeWithMyst />;
+  }
+
+  return (
+    <div className="text-sm text-slate-900 dark:text-white [&>*]:m-0 [&_a]:underline [&_a]:font-semibold">
```

Comment:
Would be good to add a `myst-` class here to enable styling.
Link this to https://github.com/jupyter-book/myst-theme/pull/667 so that we can follow up there.